config/dev.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
are you requiring this file somewhere? Are you following a guide that said you should make a config dir?